third_party/amd/lib/TritonAMDGPUTransforms/AccelerateAMDMatmul.cpp

```diff
@@ -594,10 +634,16 @@ class BlockedToWMMA : public RewritePattern {
       return failure();
 
     // get operand types
-    auto operandTypes = getOperandTypesForWmmaOp(rewriter, dotOp);
+    auto operandTypes = getOperandTypesForWmmaOp(rewriter, dotOp, wmmaVersion);
     if (operandTypes.empty())
       return failure();
 
+    // not supported yet
+    if (wmmaVersion == 2 && llvm::isa<FloatType>(operandTypes[0]) &&
+        operandTypes[0].getIntOrFloatBitWidth() == 8) {
+      return failure();
```

Comment:
Can you return a match failure here? `return rewriter.notifyMatchFailure(op, "not supported yet");` to be clear.